ArduPlane/quadplane.cpp

```diff
@@ -778,9 +778,12 @@ void QuadPlane::run_z_controller(void)
     if (now - last_pidz_active_ms > 2000) {
         // set alt target to current height on transition. This
         // starts the Z controller off with the right values
-        gcs().send_text(MAV_SEVERITY_INFO, "Reset alt target to %.1f", (double)inertial_nav.get_altitude() / 100);
-        set_alt_target_current();
-        pos_control->set_desired_velocity_z(inertial_nav.get_velocity_z());
+        float alt{};
```

Comment:
I get it, but I think many devs won't know what this does. I think that using `= 0` is a better approach.

Reply:
Yep, I need to use brace initializer everywhere ^^, I revert that !